### PR TITLE
`prettier` and `eslint` configuration

### PR DESCRIPTION
**Pull request by akshay-bharadva**
- configurations
- [x] prettier
- [x] husky
- [x] eslint

### CODE REVIEW

File: package-lock.json
Comment (palashgdev) [inline package-lock.json:1]: why this file is here ? @akshay-bharadva

File: package.json [hunk 1]
```diff
@@ -43,6 +46,21 @@
   "devDependencies": {
     "@docusaurus/module-type-aliases": "^2.3.1",
     "@tsconfig/docusaurus": "^1.0.5",
-    "typescript": "^4.6.3"
+    "@typescript-eslint/eslint-plugin": "^5.58.0",
+    "eslint": "^8.38.0",
+    "eslint-config-standard-with-typescript": "^34.0.1",
```
Comment (palashgdev) [inline package.json:51]: please make all the package exact

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx lint-staged
```
Comment (palashgdev) [inline .husky/pre-commit]: use yarn here

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (akshay-bharadva) [reply package.json:51]: made package version exact

File: .husky/pre-commit (hunk 2 above)
Comment (akshay-bharadva) [reply .husky/pre-commit]: https://typicode.github.io/husky/#/?id=install-1

**Pull request closed without merging**